bundle/manifests/kuadrant.kuadrant.io_authpolicies.yaml

```diff
@@ -2,21 +2,21 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.7.0
   creationTimestamp: null
   labels:
     app: kuadrant
-  name: authpolicies.apim.kuadrant.io
+  name: authpolicies.kuadrant.kuadrant.io
 spec:
-  group: apim.kuadrant.io
+  group: kuadrant.kuadrant.io
```

Comment:
what about having the group: `kuadrant.io` ?

`authpolicies.kuadrant.io`

`ratelimitpolicies.kuadrant.io`

`kuadrants.kuadrant.io`